tasks/main.yml

```diff
@@ -243,5 +249,13 @@
     path: /etc/systemd/system/keepalived.service.d/override.conf
     state: absent
   when:
-    - ansible_service_mgr == 'systemd'
     - not (keepalived_systemd_overrides | bool)
+
+- name: Ensuring keepalived is enabled
```

Comment:
Why not to do all-in-one just in handler?

Reply:
AFAIK, the service module is supposed to be either ensuring service state or ensuring enabled state. I had bugs in the past.
I also think it is probably better to always ensure the service is enabled, regardless of any change happening.

Reply:
I know what you're reffering to:) https://github.com/ansible/ansible/pull/68681

But it works wrong when you're trying to mask enabled service, while we're doing vice versa here. But ok, it's not something critical, just was wondering

Reply:
oh well, I was thinking of another bug tbh, far older :) glad to see there are more :p

Reply:
I have cleaned up the role of any "service" module/action plugin for now. It will be reverted when the bugs are fixed.